Mathlib/MeasureTheory/Function/Jacobian.lean

```diff
@@ -1177,8 +1177,9 @@ theorem integral_image_eq_integral_abs_det_fderiv_smul (hs : MeasurableSet s)
   rw [NNReal.smul_def, Real.coe_toNNReal _ (abs_nonneg (f' x).det)]
 
 -- Porting note: move this to `Topology.Algebra.Module.Basic` when port is over
-theorem det_one_smulRight {𝕜 : Type*} [NormedField 𝕜] (v : 𝕜) :
+theorem det_one_smulRight {𝕜 : Type*} [CommRing 𝕜] [TopologicalSpace 𝕜] [ContinuousMul 𝕜] (v : 𝕜) :
```

Comment:
Should this be moved?

Reply:
I think it should be merged with the work in https://github.com/leanprover-community/mathlib4/pull/19873 anyway, so my preference would be to leave it in place until then to make it more obvious what the generalization was.

Reply:
You mean it should be moved in #19873? Or a separate PR? Why are you worried that the generalization is not obvious? It will show up under the "Declarations diff", for example.

Reply:
I think https://github.com/leanprover-community/mathlib4/pull/19873 should probably drop this lemma entirely, and replace it with a much more general result about `smul`.

Reply:
I see that `(1 : 𝕜 →L[𝕜] 𝕜).smulRight v` is just a weird spelling of [ContinuousLinearMap.toSpanSingleton](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Algebra/Module/Basic.html#ContinuousLinearMap.toSpanSingleton) and should be changed. I don't see any proper generalization though.